app/views/admin/payout_reports/index.html.slim

```diff
@@ -20,45 +20,32 @@ table.display.table.table-bordered.table-striped
     th ID
     th Type
     th Manual
-    th Expected # of payments
-    th Total # of payments
+    th Expected - actual # of payments
     th # Payments to be paid
+    th # Legit payments missing addresses
     th Amount
-    th Fees
     th Refresh JSON
     th Download
   tbody
     - @payout_reports.each do |report|
       tr.gradeX
         td = report.created_at.strftime("%B %d, %Y, %H:%M")
-        td = link_to report.id, admin_payout_report_path(report.id)
+        td = link_to report.id[0..6], admin_payout_report_path(report.id)
         td = report.final ? "Final" : "Temp"
         td = report.manual ? "Yes" : "No"
-        td = report.expected_num_payments
-        td = report.num_payments
+        td = report.expected_num_payments - report.num_payments
```

Comment:
I think this arithmetic is evaluated... do we really want this? IMO we should do "#{report.expected_num_payments} - #{report.num_payments}"

Reply:
I care about the difference between expected and actual number of payments (if it's really high then there's a bug), but if others want to see the actual numbers, that's cool too.  I'm just trying to free up space on the table.